request.go

```diff
@@ -18,9 +19,25 @@ func (r *Request) Get(url string) (*http.Response, error) {
 		return nil, err
 	}
 
+	if len(r.headers) > 0 {
```

Comment:
This `if` is not necessary, as the `for` block will not be executed once if there are zero items in the map.

Reply:
done